### PR TITLE
Add 'rand' algorithm to speed tool

### PR DESCRIPTION
**Pull request by p-steuer**
Id like to be able to measure DRBG performance using the speed tool.

Heres a suggestion.

### CODE REVIEW

File: apps/speed.c [hunk 1]
```diff
@@ -834,6 +838,16 @@ static int CRYPTO_gcm128_aad_loop(void *args)
     return count;
 }
 
+static int RAND_bytes_loop(void *args)
+{
+    loopargs_t *tempargs = *(loopargs_t **) args;
+    unsigned char *buf = tempargs->buf;
+    int count;
+    for (count = 0; COND(c[D_RAND][testnum]); count++)
```
Comment (richsalz) [inline apps/speed.c]: blank line after declarations

**Comment by p-steuer** [comment]
added speed(1) update. i chose 'rand' to be consistent with the openssl rand tool.
other options would be 'csprng' (consistent with rand_add(3) doc) or 'drbg' (consistent with nist terminology).

File: doc/man1/speed.pod [hunk 2]
```diff
@@ -20,7 +20,8 @@ B<openssl speed>
 
 This command is used to test the performance of cryptographic algorithms.
 To see the list of supported algorithms, use the I<list --digest-commands>
-or I<list --cipher-commands> command.
+or I<list --cipher-commands> command. The global CSPRNG is denoted by
+I<rand>.
```
Comment (paulidale) [inline doc/man1/speed.pod]: _by the I<rand> algorithm name._

**Comment by paulidale** [comment]
We've got multiple different DRBGs now, might it make sense to parameterise this option to allow each to be chosen individually?

**Comment by richsalz** [comment]
For now they're just different instances of all the same algorithm, so i think parameterization is more than we need.

**Comment by richsalz** [comment]
I made the wording change to the doc and merged.  Thanks!

**Pull request closed without merging**